src/_variables.scss

```diff
@@ -0,0 +1,45 @@
+$main-background: #333;
+$starwars-color: #ffe81f;
+$nameInput-background: #3a3a3a;
+
+h1 {
+	font-size: 4.8rem;
+}
+
+.scoreboard__result {
+	font-size: 3rem;
+}
+
+h2,
+legend,
+label {
+	font-size: 1.8rem;
+}
+
+button {
+	cursor: pointer;
+}
+
+.dialog__title,
+.form__area-input-name,
+legend,
+label {
+	color: $starwars-color;
+}
+
+.dialog,
+.form__area-input-name {
+	border: 1px solid $starwars-color;
+}
+
+.computer,
+.player {
+	font-size: 2rem;
+	color: $starwars-color;
+}
+
+
+.player-score, .computer-score {
+	font-size: 2rem;
+	color: #fff;
+}
```

Comment:
To jest raczej do wyrzucenia ze zmiennych. Tutaj tylko zmienne dla kolorów, fontów, rozmiarów paddingu itd np tworzysz sobie zmienne kolorów, tworzysz sobie zmienne rozmiarow fonta np heading1: 32 heading2: 24 heading3: 18 itd, to samo dla paragrafów (np dwie wersje). Paddingi tez male (4) duze (8) wieksze (16), najwieksze (24)

To co oznaczyłem, trochę mi sie nie podoba bo dotyka zbyt wielu selektorów. dialog oraz form__area-input-name, to troche zupełnie inne komponenty a sa podpiete pod ta sama klase. Lepiej to wynieść do konkretnych plikow scss odpowiadajacych za te konkretne elementy

+ white i black tez mozna wrzucic do zmiennej, hehe to smieszne ale wazne